IdentityCore/xcconfig/identitycore__idlib__ios.xcconfig

```diff
@@ -1,2 +1,17 @@
 #include "identitycore__lib__common__ios.xcconfig"
 #include "identitycore__idlib__common.xcconfig"
+
+// Add armv7s and arm64e to standard ARCHs
+ARCHS_STANDARD = $(ARCHS_STANDARD) armv7s arm64e
```

Comment:
Why do you need to override ARCHS standard? Can we just set ARCHS for iphoneOs to $(ARCHS_STANDARD) armv7s arm64e. 

Additionally, we need a separate config file for static lib specifically so that those settings wouldn't impact other targets.

Reply:
Makes sense. I'll add a new xcconfig for static lib so that it won't impact other targets.
We can also use `ARCHS` (or other variable) instead of changing `ARCHS_STANDARD`. It just need more configurations in the file.